src/brsTypes/components/RoSGNode.ts

```diff
@@ -268,11 +268,34 @@ export class Field {
     }
 }
 
+/* Hierarchy of all node Types. Used to discover is a current node is a subtype of another node */
+const subTypeHierarchy = new Map<string, string>();
```

Comment:
Nitpick on casing, but _sub_ here is a prefix and not a distinct word so it should be _subtypeHierarchy_ just like you used _subtype_ in the comment above or _isSubtypeCheck_ below.